src/components/CountTableRow.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const CountTableRow = ({label, value, ...props }) => (
+    <tr>
+        <td>{label}:</td>
+        <td>
+            {Math.round(value[1] * 1000.0)/10}%                              
+            <span className="percent">{value[0]}</span>
```

Comment:
Probably a good idea to change this class name to "additional_detail", "number_detail" or something similar and update the "percent" analytics class name too.